test/metabase/db/env_test.clj

```diff
@@ -20,3 +20,11 @@
          clojure.lang.ExceptionInfo
          #"Unsupported application database type: \"sqlserver\""
          (#'mdb.env/connection-string->db-type "jdbc:sqlserver://bad")))))
+
+(deftest format-connection-uri-test
+  (let [conn-uri "postgresql://localhost:metabase?username=johndoe"
+        jdbc-conn-uri (str "jdbc:" conn-uri)]
+    (doseq [[input expected] [[conn-uri jdbc-conn-uri]
+                              [jdbc-conn-uri jdbc-conn-uri]
+                              [nil nil]]]
+      (is (= (#'mdb.env/format-connection-uri input) expected)))))
```

Comment:
you have these backwards, it should be `expected` and then `actual`

it only matters for formatting the error when it fails, but it definitely makes the errors confusing :)

Reply:
ah! changing it. Thanks!